package.json

```diff
@@ -26,15 +26,15 @@
   },
   "dependencies": {
     "@mongodb-js/saslprep": "^1.3.0",
-    "bson": "^7.0.0-alpha.2",
-    "mongodb-connection-string-url": "^4.0.0-alpha"
+    "bson": "^7.0.0",
+    "mongodb-connection-string-url": "^7.0.0"
   },
   "peerDependencies": {
     "@aws-sdk/credential-providers": "^3.806.0",
-    "@mongodb-js/zstd": "^3.0.0-alpha",
+    "@mongodb-js/zstd": "^3.0.0",
     "gcp-metadata": "^7.0.1",
-    "kerberos": "3.0.0-alpha",
-    "mongodb-client-encryption": "^7.0.0-alpha",
+    "kerberos": "3.0.0",
+    "mongodb-client-encryption": ">=7.0.0 <7.1.0",
```

Comment:
Just calling out this change - we do want to make this change in v7 but it isn't a simple dependency update.